public/app/features/alerting/unified/api/dataSourcesApi.ts

```diff
@@ -0,0 +1,11 @@
+import { DataSourceJsonData, DataSourceSettings } from '@grafana/data';
+
+import { alertingApi } from './alertingApi';
+
+export const dataSourcesApi = alertingApi.injectEndpoints({
```

Comment:
not sure if I like adding this to `alertingApi` when it's technically part of the Grafana API but there seems to have been some precedent with `dashboardsApi` – to refactor this later perhaps?

Reply:
I think it's ok as we are the only consumers of this api so far.

Reply:
As above. I don't think there is RTK query Grafana API object. We are one of the first teams to use it